samples/deployment/aws/deploy_and_destroy.yaml

```
@@ -0,0 +1,130 @@
+- name: "My AWS infrastructure"
+  hosts: localhost
+  gather_facts: no
+  vars:
+    ftd_build_id: "6.4.0-1304"
+    ec2_instance_type: "c4.xlarge"
+    ansible_ssh_private_key_name: "ngfwv-prod-temp-kp.pem"
+    ansible_ssh_private_key_file: "./ngfwv-prod-temp-kp.pem"
+  tasks:
+    # Prepare Env Section
+    - name: "Create prod key"
+      ec2_key:
+        name: "{{ ansible_ssh_private_key_name }}"
+      register: prod_kp
+    - name: "Save prod key"
+      copy:
+        content: "{{ prod_kp.key.private_key }}"
+        dest: "{{ ansible_ssh_private_key_file }}"
+        mode: 0600
+      when: prod_kp.changed
+    - name: "Create VPC"
```

```suggestion
    - name: "Create a VPC"
```
Please, revise task names and add articles. Especially, for those starting with `Create`.

---

fixed